### ISSUE TITLE
Empty response

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/commonsmisc#15]
For https://tools.wmflabs.org/urbanecmbot/commonsmisc/feedback.py?user=Syced here is what I receive, after a long wait time:
```
Content-type: application/json
```
I guess there is a server-side problem?
Thanks! :-)

**Comment by urbanecm** [comment]
It seems to work, but to be quite slow

```
urbanecm@notebook ~
$ time curl 'https://tools.wmflabs.org/urbanecmbot/commonsmisc/feedback.py?user=Syced'
{"imagesEditedBySomeoneElse": 451, "articlesUsingImages": 2386, "deletedUploads": 25, "featuredImages": {"Featured_pictures_on_Wikimedia_Commons": 0, "Quality_images": 0}, "user": "Syced", "status": "ok", "thanksReceived": 17, "uniqueUsedImages": 318}

real    1m2.267s
user    0m0.016s
sys     0m0.047s
```

**Comment by maskaravivek** [comment]
It is happening for me as well. 

<img width="1440" alt="Screen Shot 2019-06-05 at 3 05 22 PM" src="https://user-images.githubusercontent.com/3069373/58946392-5eb74980-87a3-11e9-8c36-1a3547421429.png">

**Comment by maskaravivek** [comment]
Same happens while using curl. 

<img width="942" alt="Screen Shot 2019-06-05 at 3 06 21 PM" src="https://user-images.githubusercontent.com/3069373/58946449-81e1f900-87a3-11e9-9050-1481a6833c9e.png">

**Comment by urbanecm** [comment]
Can reproduce now... Interesting.

**Comment by nicolas-raoul** [comment]
I tried a few times again, still the same lone `Content-type: application/json` for me too.

**Comment by urbanecm** [comment]
Seems there was a schema change I missed, fixed by bca25f46c2499e18836ccb41c5c8a830d9f58c2c and eb8792c62c7c54343bb5cf2f45f4fbd7fece53ef.

**Comment by nicolas-raoul** [comment]
Thanks a lot for your fast reaction, it seems to work now :-)
